### PR TITLE
Explain how to connect to the server with the CLI running in a container

### LINKED ISSUE #13269: Explain how to connect to the server with the CLI running in a container
### Description

 Explain how to connect to the server with the CLI running in a container

### Implementation ideas

_No response_

### PR DESCRIPTION
**Pull request by karesti**
Closes #13269

### CODE REVIEW

File: documentation/src/main/asciidoc/stories/assembly_cli_starting.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@
 = Getting Started with {brandname} CLI
 The command line interface (CLI) lets you remotely connect to {brandname} Server to access data and perform administrative functions.
 Complete the following procedures to learn basic CLI usage such as creating users, connecting to {brandname}, and navigating resources.
-
+include::{topics}/proc_cli_use_container.adoc[leveloffset=+1]
```
Comment (pruivo) [inline documentation/src/main/asciidoc/stories/assembly_cli_starting.adoc:6]: I don't think you can create users with the container (the CLI requires access to the server filesystem to create the user). Should the procedure be added to `proc_cli_connecting.adoc` instead?

Comment (pruivo) [reply documentation/src/main/asciidoc/stories/assembly_cli_starting.adoc:6]: PS. We probably need an example of how to connect with an SSL-enabled server.

Comment (karesti) [reply documentation/src/main/asciidoc/stories/assembly_cli_starting.adoc:6]: I'm going to add more info about this. 
concerning SSL, that should definitely be covered in the SSL part

Comment (karesti) [reply documentation/src/main/asciidoc/stories/assembly_cli_starting.adoc]: So, I've checked all, and I moved this to the section of connecting. 
when we run the server in a container, we need to pass the admin and password credentials or some will be created for us. unless we override the server config. I don't think we need to add anything else here. however, I think we need to report the docs of the images to this documentation

File: documentation/src/main/asciidoc/topics/proc_cli_connecting_container.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
+[id='cli_start-{context}']
+= Connecting to {brandname} Servers running in a container
+Establish CLI connections to {brandname} when running in a container.
```
Comment (theashiot) [inline documentation/src/main/asciidoc/topics/proc_cli_connecting_container.adoc:3]: I suggest either:

You can establish a connection with a {brandname} server running in a container through the command line interface (CLI) by following this procedure.

or

You can access the command line interface (CLI) of a {brandname} server running in a container by following this procedure.

File: documentation/src/main/asciidoc/topics/proc_cli_connecting_container.adoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
+[id='cli_start-{context}']
+= Connecting to {brandname} Servers running in a container
```
Comment (theashiot) [inline documentation/src/main/asciidoc/topics/proc_cli_connecting_container.adoc:2]: I suggest either:

Establishing a connection with {brandname} running in a container through the CLI  

or

Accessing the CLI of  {brandname}  running in a container

File: documentation/src/main/asciidoc/topics/proc_cli_connecting_container.adoc [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+Establish CLI connections to {brandname} when running in a container.
+
+.Prerequisites
+
+Have at least one running {brandname} server instance in a container.
+
+.Procedure
+
+. Open a terminal
```
Comment (theashiot) [inline documentation/src/main/asciidoc/topics/proc_cli_connecting_container.adoc:11]:
```suggestion
. Open a terminal.
```

File: documentation/src/main/asciidoc/topics/proc_cli_connecting_container.adoc [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+. Start the CLI.
++
+* *Podman or Docker:*
++
+[source,options="nowrap",subs=attributes+]
+----
+podman/docker run -it --net=host infinispan/cli
+----
+
+. Run the [command]`connect` command and enter your username and password when prompted
```
Comment (theashiot) [inline documentation/src/main/asciidoc/topics/proc_cli_connecting_container.adoc:21]:
```suggestion
. Run the `connect` command to connect  to a {brandname} server running on the default port of `11222`. Enter your username and password when prompted. 
```
I'm not sure if the "[command]"  is portable to different asciidoctor versions. If you know it is, pleas retain it!

File: documentation/src/main/asciidoc/topics/proc_cli_connecting_container.adoc [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
++
```
Comment (theashiot) [inline documentation/src/main/asciidoc/topics/proc_cli_connecting_container.adoc:22]:
```suggestion
```

File: documentation/src/main/asciidoc/topics/proc_cli_connecting_container.adoc [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+* {brandname} Server on the default port of `11222`:
```
Comment (theashiot) [inline documentation/src/main/asciidoc/topics/proc_cli_connecting_container.adoc:23]:
```suggestion
```

**Review by theashiot — COMMENTED** [review]
Thanks @karesti for the docs! They look good. I have added some suggestions inline for your consideration.

**Comment by andyuk1986** [comment]
@karesti looks good to me.

**Comment by karesti** [comment]
@theashiot  I reported all the changes

**Review by pruivo — APPROVED** [review]
It looks good to me. It requires a GHI.

**Comment by karesti** [comment]
@pruivo issue added and commit message updated

**Comment by pruivo** [comment]
Thanks @karesti 
@andyuk1986 @theashiot can I get a green light to merge this?

**Comment by karesti** [comment]
@pruivo Anna gave her green light a while ago and I reported all the changes

**Comment by pruivo** [comment]
merged! thanks @karesti !

**Pull request merged**